finesse/gui/main_window.py

```diff
@@ -69,7 +69,7 @@ def __init__(self) -> None:
         spectrometer: QGroupBox = SpectrometerControl()
 
         # Setup for interferometer monitor
-        em27_monitor = EM27Monitor()
+        em27_monitor = SensorsPanel()
         em27_monitor.setSizePolicy(
```

Comment:
Shouldn't this be renamed something different than `em27_monitor`?

Reply:
Indeed!